src/widgets/results_widgets/a_priori_results.py

```diff
@@ -1,16 +1,28 @@
-from PyQt5.QtWidgets import QWidget, QGroupBox, QFormLayout, QLabel, QTableView, QVBoxLayout
+from typing import List
 
+import matplotlib.pyplot as plt
+import pandas as pd
+from PyQt5.QtWidgets import QWidget, QGroupBox, QFormLayout, QLabel, QTableView, QVBoxLayout, QHBoxLayout
+
+from visualization.associations import APrioriScatterPlot, APrioriGraphPlot, APrioriGauge
 from widgets import QtTable
 
 
 class APrioriResultsWidget(QWidget):
```

Comment:
WDYT to add information about that user can click on table to show more visualization? I think that this text should be in place where are the plots after clicking.

Reply:
The second thing is add possibility to click on indexes in table, we discuss about it on messenger.